kinder/ci/tools/update-workflows/cmd/main.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	"k8s.io/kubeadm/kinder/ci/tools/update-workflows/pkg"
+)
+
+func main() {
+	// prepare flags
+	flags := pkg.NewFlags()
+	flag.Var(&flags.KubernetesVersion, "kubernetes-version", "Kubernetes version (e.g. v1.21)")
+	flag.StringVar(&flags.PathConfig, "config", "", "config file")
+	flag.StringVar(&flags.PathTestInfra, "path-test-infra", "", "path to the directory with test-infra kubeadm jobs")
+	flag.StringVar(&flags.PathWorkflows, "path-workflows", "", "path to the directory with kinder workflows")
+	flag.StringVar(&flags.ImageTestInfra, "image-test-infra", "", "image tag to use for test-infra jobs")
+	flag.IntVar(&flags.SkewSize, "skew-size", 3, "size of the Kubernetes support skew")
```

Comment:
yes, absolutely. because we actually have variadic skew.

currently we are testing 1.21, 1.20, 1.19, 1.18 so the skew is 4
once 1.21.1 is out we drop 1.18 and the skew becomes 3.

Reply:
What about documenting this as a "number of older Kubernetes minor version to be tested starting from --kuberentes-version (included)"

Reply:
SGTM, for clarity.